### PR TITLE
Add EIP-1238.md

### PR DESCRIPTION
**Pull request by TimDaub**
This is a "DRAFT" proposal on how #1238 could be implemented.

### CODE REVIEW

**Comment by SamWilsn** [comment]
Is this ready for editor review? If not, would you mind marking as a draft?

**Comment by TimDaub** [comment]
For me it's fine moving the proposal to the next stage, but e.g. I don't know if e.g. the OP of #1238 is OK with me DRAFTing and sending this document. 

But I guess they could object or give feedback in the editor stage too, right?

If we can move it forward, let's do it.

File: EIPS/eip-1238.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+---
+eip: 1238
```
Comment (MicahZoltu) [inline EIPS/eip-1238.md:2]:
```suggestion
eip: 4966
```

File: EIPS/eip-1238.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+title: Non-Transferrable Non-Fungible Tokens (Soulbound Tokens)
```
Comment (MicahZoltu) [inline EIPS/eip-1238.md:3]:
```suggestion
title: Non-Transferrable Non-Fungible Tokens (Soulbound Tokens)
description: A standard interface for non-transferrable non-fungible tokens, also known as "soulbound tokens" (short "SBT").
```

File: EIPS/eip-1238.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+author: Nicola Greco, Tim Daubenschütz <tim@daubenschuetz.de>
+discussions-to: https://github.com/ethereum/eips/issues/1238
```
Comment (MicahZoltu) [inline EIPS/eip-1238.md:5]: This needs to be a link to a thread on Ethereum Magicians.

File: EIPS/eip-1238.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+status: Draft
+created: 2022-04-01
+requires: 165, 721
+---
+
+## Simple Summary
+
+A standard interface for non-transferrable non-fungible tokens, also known as
+"soulbound tokens" (short "SBT").
+
```
Comment (MicahZoltu) [inline EIPS/eip-1238.md:17]:
```suggestion
```
This section is no longer part of EIPs.

File: EIPS/eip-1238.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
+---
+eip: 1238
+title: Non-Transferrable Non-Fungible Tokens (Soulbound Tokens)
+author: Nicola Greco, Tim Daubenschütz <tim@daubenschuetz.de>
```
Comment (MicahZoltu) [inline EIPS/eip-1238.md:4]:
```suggestion
author: Nicola Greco, Tim Daubenschütz (TimDaub), Tim Daubenschütz <tim@daubenschuetz.de>
```
```suggestion
author: Nicola Greco, Tim Daubenschütz (TimDaub)
```
At least one author needs a GitHub name.  Either of the above are acceptable.  I'm not sure if the bot will allow an author with neither GitHub nor email...

File: EIPS/eip-1238.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+
+## Backwards Compatibility
+
+We have adopted the `ERC165` and `ERC721Metadata` functions. `ERC1238`'s
+`boundTo` function is the pendant to `ERC721`'s `ownerOf` function.
+
+## Test Cases
+
+See "Implementations" section.
+
```
Comment (MicahZoltu) [inline EIPS/eip-1238.md:130]:
```suggestion
```
This section is optional for ERCs, so if you don't have content leave it out.

File: EIPS/eip-1238.md [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+## Implementations
+
+- [rugpullindex/ERC1238](https://github.com/rugpullindex/ERC1238) is a minimal
+  implementation using `burn` and `mint` and test cases released under GNU
+  General Public License v3.0.
+
```
Comment (MicahZoltu) [inline EIPS/eip-1238.md:136]: This is not a valid EIP section.  There is a reference implementation section, but it should include a barebones reference implementation inline or in `../assets/eip-4966/*`.

File: EIPS/eip-1238.md [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+1. [ERC-165](./eip-165.md) Standard Interface Detection.
+1. [ERC-721](https://github.com/ethereum/EIPs/issues/721) Token Standard.
+1. Soulbound. https://vitalik.ca/general/2022/01/26/soulbound.html
+1. RFC 2119 Key words for use in RFCs to Indicate Requirement Levels. https://www.ietf.org/rfc/rfc2119.txt
+
+**Issues**
+
+1. The Original ERC-1238 Issue. https://github.com/ethereum/eips/issues/1238
+
```
Comment (MicahZoltu) [inline EIPS/eip-1238.md:149]: This isn't a valid EIP section.  For links to other EIPs, please inline them using relative link format like `[EIP-721](./eip-721.md)`.  External links should not be included.

**Review by MicahZoltu — CHANGES_REQUESTED** [review]

**Comment by MicahZoltu** [comment]
I'm closing all PRs that were open on April 1st since we received a number of "joke" EIPs.  If this was in fact a serious EIP that you would like to have reviewed, please resubmit.

Note: Please keep in mind that EIP editors are all volunteers and have more work than time available.

File: EIPS/eip-1238.md (hunk 1 above)
Comment (TimDaub) [reply EIPS/eip-1238.md:2]: Why suggest 4966? I wanted this PR to provide a solution for the discussion and post in #1238. Is that possible?

**Comment by TimDaub** [comment]
> If this was in fact a serious EIP that you would like to have reviewed, please resubmit.

It's not a joke.

> Note: Please keep in mind that EIP editors are all volunteers and have more work than time available.

I'm also not getting any money for writing this.

**Comment by TimDaub** [comment]
Continued in https://github.com/ethereum/EIPs/pull/4973

File: EIPS/eip-1238.md (hunk 1 above)
Comment (MicahZoltu) [reply EIPS/eip-1238.md:2]: I would be amenable to using that number if you can get approval from the original author of that issue.

Comment (TimDaub) [reply EIPS/eip-1238.md:2]: @nicola, I've been busy with defining a standard for your issue at #1238 over at #4973. And so far I've kept you as the co-author. Would you be willing to accept the documents attached to #4973 as the official ERC1238 standard? If not, we'll move forward with ERC4973. Please also let me know if you're interested in being a co-author. If not I'll remove you of course. Thank you.

**Pull request closed without merging**